app/main/forms/frameworks.py

```diff
@@ -0,0 +1,31 @@
+from flask.ext.wtf import Form
+from wtforms import validators, StringField
+from wtforms import BooleanField as _BooleanField
+
+
+class BooleanField(_BooleanField):
+    """Tri-state boolean field
```

Comment:
Yeah I was only joking.  Three-valued booleans are a totally sane thing for us to have and there's no way it could lead to confusion.